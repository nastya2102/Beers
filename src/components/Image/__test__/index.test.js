import React from 'react'
import { render } from '@testing-library/react'
import Image from '../index'

describe('Image', () => {
  test('should render Image component', () => {
    const { container } = render(<Image />)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Image component', () => {
    const { container } = render(<Image />)

    expect(container).toMatchSnapshot()
  })
})
