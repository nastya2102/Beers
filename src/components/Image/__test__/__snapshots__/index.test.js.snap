// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should match snapshot Image component 1`] = `
<div>
  <div
    class="sc-bdfBwQ hVBrXR"
  />
</div>
`;
